### PR TITLE
add threadpool timer state spec

### PR DESCRIPTION
**Pull request by M-iceberg**

### CODE REVIEW

File: win32/devdoc/threadpool_win32.md [hunk 1]
```diff
@@ -215,6 +215,8 @@ MOCKABLE_FUNCTION(, int, threadpool_timer_restart, TIMER_INSTANCE_HANDLE, timer,
 
 **SRS_THREADPOOL_WIN32_42_019: [** If `timer` is `NULL`, `threadpool_timer_restart` shall fail and return a non-zero value. **]**
 
+If timer is `DESTROYING`, `threadpool_timer_restart` shall fail and return a non-zero value.
```
Comment (jebrando) [inline win32/devdoc/threadpool_win32.md:218]: >If timer is [](http://example.com/codeflow?start=0&length=11)

I believe that this should read if timer state is DESTROYING ...

**Review by jebrando — APPROVED** [review]
:shipit:

File: win32/devdoc/threadpool_win32.md (hunk 1 above)
Comment (M-iceberg) [reply win32/devdoc/threadpool_win32.md]: fixed

**Review by jebrando — APPROVED** [review]
:shipit:

**Pull request merged**